CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Enhancements and minor changes
 - Added support for NWB schema 2.7.0.
   - ...
+  - ...
```

Comment:
Not sure what the "- ..." are for

Reply:
I was hoping to avoid merge conflicts in the changelog by editing different `- ...` in different PRs. It kind of worked but not really... This will get cleaned up before merging the nwb-schema-2.7.0 branch to dev.